corehq/sql_accessors/migrations/0040_fix_case_reindex_query.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import RawSQLMigration, HqRunSQL
+
+migrator = RawSQLMigration(('corehq', 'sql_accessors', 'sql_templates'), {})
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sql_accessors', '0038_save_case_deletion_fields'),
+    ]
+
+    operations = [
+        HqRunSQL(
+            "DROP FUNCTION IF EXISTS get_all_cases_modified_since(timestamp with time zone, integer)",
```

Comment:
Noticed that this was still in the prod DB. Must have forgotten to remove it when the signature changed.